src/widgets/radio.jsx

```diff
@@ -49,88 +194,62 @@ var BaseRadio = React.createClass({
+                                var shouldToggle =
+                                    this.props.multipleSelect ||
+                                    this.props.deselectEnabled;
+                                this.checkOption(
+                                    i,
+                                    shouldToggle ? !choice.checked : true);
                             }
-                            onClick={!this.props.labelWrap ? null : (e) => {
-                                // Don't send this to the scratchpad
-                                e.preventDefault();
-                                if (!this.props.apiOptions.readOnly) {
-                                    var shouldToggle =
-                                        this.props.multipleSelect ||
-                                        this.props.deselectEnabled;
-                                    this.checkOption(
-                                        i,
-                                        shouldToggle ? !choice.checked : true);
-                                }
-                            }}>
-                    <div>
-                        <span className="checkbox">
-                            <input
-                                ref={"radio" + i}
-                                type={inputType}
-                                name={radioGroupName}
-                                checked={choice.checked}
-                                onClick={(e) => {
-                                    // Avoid sending this to the parent
-                                    e.stopPropagation();
-                                }}
-                                onChange={(e) => {
-                                    this.checkOption(i, e.target.checked);
-                                }}
-                                disabled={this.props.apiOptions.readOnly} />
-                        </span>
-                        {/* A pseudo-label. <label> is slightly broken on iOS,
-                            so this works around that. Unfortunately, it is
-                            simplest to just work around that everywhere. */}
-                        <span
-                                className={
-                                    ApiClassNames.RADIO.OPTION_CONTENT + " " +
-                                    ApiClassNames.INTERACTIVE
-                                }
-                                style={{
-                                    cursor: "default",
-                                }}>
-                            {content}
-                        </span>
-                        {showClue &&
-                            <div className="perseus-radio-clue">
-                                {choice.clue}
-                            </div>}
-                    </div>
-                </li>;
+                    }}>
 
+                    {React.createElement(element, elementProps)}
```

Comment:
If you rename `element` to `Element`, then this can be written as

    <Element {...elementProps} />

React decides whether something in jsx is a DOM literal or a user defined react class based on whether it starts with a capitol letter, so we need to rename `element` to get this to actually work.